tests/lambda_endpoints/test_download_query_api.py

```diff
@@ -0,0 +1,80 @@
+from pathlib import Path
+
+import pytest
+
+from sds_data_manager.lambda_code.SDSCode.download_query_api import lambda_handler
+
+BUCKET_NAME = "test-bucket"
+TEST_FILE = "science_block_20221116_163611Z_idle.bin"
+
+
+@pytest.fixture(autouse=True)
+def setup_s3(s3_client):
```

Comment:
I'm not sure how much we should care about this for test functions, but it looks like some docstrings are missing here (as well as the `test_object_exists_with_s3_uri` function)

Reply:
Yep, good catch here! I think if something isn't immediately obvious we should have a docstring or comment letting us know what it is. I've also added a few others.

Specifically, though, I'd say that fixtures definitely should have some comments since they are meant for re-use, tests I don't mind as much if they don't have docstrings as long as they are named well and obvious about what they do. So, up to the developer on those.

Reply:
I'd also like to see a specific note on whether the tests are using mocked clients or real ones - unless it's obvious. So I think the existing comment is good!